watertap/unit_models/MD/membrane_distillation_0D.py

```diff
@@ -0,0 +1,567 @@
+#################################################################################
+# WaterTAP Copyright (c) 2020-2023, The Regents of the University of California,
+# through Lawrence Berkeley National Laboratory, Oak Ridge National Laboratory,
+# National Renewable Energy Laboratory, and National Energy Technology
+# Laboratory (subject to receipt of any required approvals from the U.S. Dept.
+# of Energy). All rights reserved.
+#
+# Please see the files COPYRIGHT.md and LICENSE.md for full copyright and license
+# information, respectively. These files are also available online at the URL
+# "https://github.com/watertap-org/watertap/"
+#################################################################################
+
+# Import Pyomo libraries
+from pyomo.environ import Var, NonNegativeReals, value, Constraint
+from pyomo.common.config import Bool, ConfigDict, ConfigValue, ConfigBlock, In
+from idaes.core import FlowDirection
+from pyomo.environ import (
+    NonNegativeReals,
+    Var,
+    check_optimal_termination,
+    exp,
+    units as pyunits,
+)
+
+from .MD_channel_base import (
+    ConcentrationPolarizationType,
+    TemperaturePolarizationType,
+    MassTransferCoefficient,
+    PressureChangeType,
+    FrictionFactor,
+)
+from idaes.core import (
+    FlowDirection,
+    EnergyBalanceType,
+    MaterialBalanceType,
+    MomentumBalanceType,
+    useDefault,
+)
+
+from idaes.core import declare_process_block_class
+from idaes.core.util import scaling as iscale
+from .MD_channel_base import (
+    TemperaturePolarizationType,
+    MassTransferCoefficient,
+    PressureChangeType,
+)
+from .MD_channel_0D import MDChannel0DBlock
+from .membrane_distillation_base import (
+    MembraneDistillationBaseData,
+)
+from idaes.core.util.config import is_physical_parameter_block, DefaultBool
+
+# __author__ = "Elmira Shamlou"
+
+
+@declare_process_block_class("MembraneDistillation0D")
+class MembraneDistillationData(MembraneDistillationBaseData):
+    """
+    Standard DCMD Unit Model Class:
+    - zero dimensional model
+    - steady state only
+    """
+
+    CONFIG = ConfigBlock()
+    CONFIG.declare(
+        "dynamic",
+        ConfigValue(
+            default=False,
+            domain=In([False]),
+            description="Dynamic model flag - must be False",
+            doc="""Indicates whether this model will be dynamic or not.
+    **default** - False. Membrane units do not yet support dynamic
+    behavior.""",
+        ),
+    )
+    CONFIG.declare(
+        "has_holdup",
+        ConfigValue(
+            default=False,
+            domain=In([False]),
+            description="Holdup construction flag - must be False",
+            doc="""Indicates whether holdup terms should be constructed or not.
+    **default** - False. Membrane units do not have defined volume, thus
+    this must be False.""",
+        ),
+    )
```

Comment:
Any reason that we need `dynamic` and `has_holdup` in the main config block as well as in the config template for hot and cold channels? I would think you could eliminate these 2 from config_template, but maybe there is some reason for doing this.